islandora_checksum_checker.api.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * This file documents hook functions for the Islandora Checksum Checker module.
+ */
+
+/**
+ * Notify modules of the outcome of the checksum validation.
+ *
+ * @param string $pid
+ *   The current object's PID.
+ * @param string $dsid
+ *   The current datastream's DSID.
+ * @param array $ds_info
+ *   Output of Tuque's APIM getDatastream() method, with 'validateChecksum'
+ *   set to TRUE. See islandora_checksum_checker_validate_checksum() for
+ *   details on the content of this array.
+ * @param string $outcome
+ *   One of 'valid', 'invalid', or 'disabled'.
+ * @param array $exceptions
```

Comment:
Could be useful to document that the array is actually of php exception objects (class derived), could be done via a code example where each exceptions is itereated and ::getMessage() method is invoked or simply written as part of the "@param" description

Reply:
Good point, I'll update the @param description to indicate it's an array of exception objects.

Reply:
Thanks. The rest looks pretty good. Will give it a spin/ and if all goes well(for sure will) approve and merge this week. Nice improvement

Reply:
OK, change pushed.